config/game_options.txt

```diff
@@ -394,6 +394,7 @@ ROUNDSTART_RACES human
 ROUNDSTART_RACES lizard
 #ROUNDSTART_RACES fly
 ROUNDSTART_RACES plasmaman
+ROUNDSTART_RACES moth
```

Comment:
OFF

Reply:
why? they're no better than humans, they just look stupid

Reply:
# O F F

Reply:
do what the maintainer says man.

Reply:
off

Reply:
Dont need more cosmetic roundstart races, these can be silly things from xenobio or jungle dweller ghost role or something